scripts/db-heal-test.py

```diff
@@ -0,0 +1,217 @@
+import os
+import subprocess
+import tempfile
+import time
+import shutil
+from pathlib import Path
+import sys
+import flag
+
```

Comment:
the script should have been documented: what it does, how to run it, what prerequisites must be met (python packages installation, etc) .

Reply:
I really insists on having some documentation here. I had again hard time to start the script, it printed many errors first, then I realised it must be run from the `script` directory - this is nowhere documented, how can one know it?

Furthermore, running the script from the root directory is IMHO more intuitive than running from any subdirectories. It is more straightforward to integrate in further scripts like pipelines - one just checkouts the repository and executes the script, no further `cd` is needed.